platformio.ini

```diff
@@ -524,6 +524,29 @@ board_build.f_flash = 80000000L
 board_build.flash_mode = qio
 monitor_filters = esp32_exception_decoder
 
+[env:esp32s3dev_16MB_PSRAM_opi]
+board = esp32-s3-devkitc-1
+board_build.arduino.memory_type = qio_opi
+platform = ${esp32s3.platform}
+platform_packages = ${esp32s3.platform_packages}
+upload_speed = 921600
+framework = arduino
+
+# Configure options for the N16R8V variant
+build_flags= ${common.build_flags} ${esp32s3.build_flags}
+  -D CONFIG_LITTLEFS_FOR_IDF_3_2 -D WLED_WATCHDOG_TIMEOUT=0
+  -D ARDUINO_USB_CDC_ON_BOOT=0  -D ARDUINO_USB_MODE=1 ;; for boards with serial-to-USB chip
+  ;-D ARDUINO_USB_CDC_ON_BOOT=1 -D ARDUINO_USB_MODE=1      ;; for boards with USB-OTG connector only (USBCDC or "TinyUSB")
+  ; -D WLED_RELEASE_NAME=ESP32-S3_PSRAM
+  -D WLED_USE_PSRAM -DBOARD_HAS_PSRAM ; tells WLED that PSRAM shall be used
+  -D BOARD_HAS_PSRAM
```

Comment:
this line is a duplicate of the previous `-D WLED_USE_PSRAM -DBOARD_HAS_PSRAM`

Reply:
Changed to use extends